demo.sh

```diff
@@ -526,6 +523,15 @@ help() {
     echo "    -v --verbose  enable verbose output"
 }
 
+base64_file_nowrap() {
+    # This function encodes the specified file to base64 format without wrapping lines.
+    # By default, Linux systems wrap base64 output every 76 columns. We use 'tr -d' to remove whitespaces.
+    # Note that the option '-w0' for 'base64' doesn't work on Mac OS X due to different flags.
+    # Arguments:
+    #   $1: File path to be encoded
+    cat $1 | base64 | tr -d '\r\n'
```

Comment:
please quote the argument to avoid expansion issues

```suggestion
    cat "$1" | base64 | tr -d '\r\n'
```